deb/libsoletta/control

```diff
@@ -0,0 +1,34 @@
+Source: libsoletta
+Priority: optional
+Maintainer: Anselmo L. S. Melo <anselmo.melo@intel.com>
+Build-Depends: debhelper (>= 8.0.0), cdbs, python3 (>=3.4), chrpath, python3-jsonschema, libudev-dev, libpcre3-dev, libicu-dev, libcurl4-openssl-dev
```

Comment:
Makes sense.
We can't continue relying on simply running alldefconfig to generate the .config for packages, fine tuning is needed.